extensions/vscode/webviews/homeView/vite.config.ts

```diff
@@ -39,13 +39,6 @@ export default defineConfig({
     environment: "jsdom",
     coverage: {
       enabled: true,
-      thresholds: {
-        functions: 30.13,
-        lines: 17.46,
-        branches: 44.82,
-        statements: 17.46,
-        autoUpdate: true,
-      },
```

Comment:
```
      thresholds: {
        functions: 30,
        lines: 17,
        branches: 44,
        statements: 17,
        autoUpdate: false,
      },
```

Reply:
That earlier failure I referenced above occurred when I attempted to do this myself, with autoupdate enabled:

https://github.com/posit-dev/publisher/actions/runs/12777773860/job/35619266385?pr=2524

Reply:
`autoUpdate` has been set to false, thanks for the suggestion.